app/views/ListView.scala.html

```diff
@@ -26,7 +26,7 @@
 )
 @(form: Form[Boolean], viewmodel: FormPageViewModel[ListViewModel])(implicit request: Request[_], messages: Messages)
 
-    @layout(pageTitle = title(form, if(viewmodel.optViewOnlyDetails.nonEmpty) viewmodel.optViewOnlyDetails.get.title.toMessage else viewmodel.title.toMessage),showBackLink = viewmodel.showBackLink) {
+    @layout(pageTitle = title(form, if(viewmodel.optViewOnlyDetails.nonEmpty) viewmodel.optViewOnlyDetails.get.title.toMessage else viewmodel.title.toMessage), showBackLink = viewmodel.showBackLink) {
```

Comment:
Title (and heading) should be different in pre-population mode. Can be set up by the controller and just passed in as `viewmodel.title` value though.